util/coinach.ts

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env node
+
+// Helper for automating SaintCoinach -- https://github.com/ufx/SaintCoinach
+
+import { promisify } from 'util';
+import fs from 'fs';
+import path from 'path';
+import { exec } from 'child_process';
+
+
+const _COINACH_EXE = 'SaintCoinach.Cmd.exe';
```

Comment:
Do you think it's be worth converting these to camelCase names? The uppercase constants is a Python PEP8 convention.

Reply:
I would expect to follow the convention set by [typescript-eslint/naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/v4.9.1/packages/eslint-plugin/docs/rules/naming-convention.md), which would accept `coinachExe` (like you suggested), or `COINACH_EXE` (what exists without the leading underscore). Throwing my personal opinion in, I like either format but I would consider `coinachExe` to be more correct given the usage is constrained solely to this class.